schemas/lifeCycleData.json

```diff
@@ -0,0 +1,214 @@
+              "type": "object",
+              "properties": {
+                "renewables": {
+                  "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                  "description": "Renewables"
+                },
+                "renewablesSustainablyProduced": {
+                  "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                  "description": "Renewables"
+                },
+                "notRenewables": {
+                  "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                  "description": "Renewables"
+                },
+                "recycled": {
+                  "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                  "description": "Renewables"
+                },
+                "reused": {
+                  "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                  "description": "Renewables"
+                },
+                "recyclingEfficiency": {
+                  "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                  "description": "Renewables"
+                }
+              },
+              "additionalProperties": false,
+              "minProperties": 1,
+              "required": []
+            },
+            "output": {
+              "title": "Output",
+              "type": "object",
+              "properties": {
+                "availableForRecycle": {
+                  "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                  "description": "Renewables"
+                },
+                "recyclingEfficiency": {
+                  "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                  "description": "Renewables"
+                },
+                "landfill": {
+                  "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                  "description": "Renewables"
+                },
+                "incineration": {
+                  "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                  "description": "Renewables"
+                },
+                "availableForReuse": {
+                  "$ref": "number.json#/$defs/percentDecimalBetweenZeroAndOne",
+                  "description": "Renewables"
+                }
+              },
+              "additionalProperties": false,
+              "minProperties": 1,
+              "required": []
+            },
+            "detachability": {
+              "title": "Output",
+              "type": "object",
+              "properties": {
+                "connectionType": {
+                  "type": "string",
+                  "enum": [
+                    "Unknown",
+                    "HardChemicalConnection",
+                    "SoftChemicalConnection",
+                    "DirectConnection",
+                    "AddedConnectionConnection",
```

Comment:
Is this a typo?

Reply:
@pascalkeppler I think we need a breaking change sooner or later. Let's discuss it in our next meeting how this can be done.